### ISSUE TITLE
Consider adding "tree-style" view in new bookmarks sidebar

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#34847]
## Description 

Many of the users who are giving feedback about the new bookmarks sidebar layout/functionality appear to have two main issues with the changes — the order of the bookmarks that they have set (potentially meticulously) over time is not retained (see https://github.com/brave/brave-browser/issues/34776) and that they can no longer change the visual style to the "tree-view". 

The visual and list style options are nice, but they do require additional clicks (potentially several, depending on the number of bookmarks)/interactions to navigate to the desired bookmark/folder, while tree-style view doesn't require going up/down any levels as everything is visible on the same screen.

I am unclear as to just how much work it would take to add/maintain a change like this but I think we will certainly see enough requests for this as more users are exposed to the feature to justify at least considering implementing it.

cc @rebron @simonhong

**Comment by angsama** [comment]
The Bookmark Sidebar lost the productivity it had.Sure the new option to organise as you want is good but destroyed the old style. It should have the ability to maintain the bookmark order as the user set. Like in the old style or in the bookmark settings menu. 
The classic "hierarchy-tree" type style with folders, is a model that it is tested and more workflow efficient if the user have too many bookmarks.

-Another problem is the visibility of the list. The text is small and you can't see your bookmarks if you have many.
-The +new folder button is to close to the top bar when you open a bookmark folder. You can miss click easily. 
-The new sidebar bookmark doesn't "remember" the last position-folder. 
-Too many clicks to reach your destined folder. 

Thanks for the hard work. I hope more users give more suggestions.

**Comment by mromanek** [comment]
I'm sure someone prefers the new functionality, but I've gotten a deluge of tickets from users who are quickly abandoning Brave because of this change.  Most of my users have deep trees with only a handful of bookmarks at each level, mostly relateing to top-level links and "quicklinks" to sub-level detail links. 

Having a tree view back would surely spread Christmas cheer. :)

**Comment by DatuStram** [comment]
Yes please, for the love of everything that's good in this world, add a simple "tree-view" style to the new bookmarks sidebar.

As someone who's gathered thousands of bookmarks over the years and organize them in a deep and hierarchized tree of folders, the new bookmarks sidebar has completely destroyed my workflow, and I'm sure I'm not the only one.

Right now, I have to rely on an external extension [Bookmark Sidebar](https://chromewebstore.google.com/detail/bookmark-sidebar/jdbnofccmhefkmjbkkdkfiicjkgofkdh) to get this functionality back, which defeats the purpose of having a built-in functionality.

**Comment by Doxorn** [comment]
Yes please add back tree style view. Why was this half baked update even pushed to the non-nightly brave users? Keep beta code in beta.
With the current functionality of the sidebar I can't use it at all. While I don't have many bookmarks in the sidebar, I need to see multiple open folders' content at once. That is why I loved the previous sidebar version.

**Comment by DatuStram** [comment]
> Yes please add back tree style view. Why was this half baked update even pushed to the non-nightly brave users? Keep beta code in beta. With the current functionality of the sidebar I can't use it at all. While I don't have many bookmarks in the sidebar, I need to see multiple open folders' content at once. That is why I loved the previous sidebar version.

It seems this "new and improved" (😒) bookmarks sidebar is a direct copy/paste from chrome itself, since ungoogled chromium and other browsers have to endure this blight as well…

I'm still clueless as to who in their right mind would think it'd be a great change though:

- Like you said, it doesn't allow viewing multiple open folders concurrently.
- It doesn't remember which folders were open when you close and reopen it.
- _It has no way of displaying the bookmarks as the user organized them_, which is pretty much a thing since the ol'Netscape Navigator!

I really hope the Brave team can come to the rescue and implement a simple, no fuss, functional bookmarks sidebar again, because right now their browser is a real pain to use 😣

**Comment by Brave-Matt** [comment]
Another +1:
https://community.brave.com/t/bookmark-folders-drop-down/524671/6

**Comment by techeddie** [comment]
please bring back the hierarchy tree view. it was easier and comfortable to work with multiple open folders. the new changes does only slow down the workflow. thank you

**Comment by JoshuaS3** [comment]
+1. This functionality is pretty crucial to my workflow. Automatic update just nuked it with few settings for customization. Switching to Firefox until this is fixed.

**Comment by prime4lpha** [comment]
We need the tree view back. This is unusable for me. I have various bookmarks organised in different folders and this is horrific to look at, let alone find anything quickly. The whole point of these bookmarks is easy management and access of important links. This new style is anything but that.

I don't even understand who thought this would be a good idea.

**Comment by fortizc** [comment]
+1 the previous tree view was awesome.

**Comment by djagoo** [comment]
+1 for bringing back the tree view, this is absolute unusable for me
